### PR TITLE
Update react, remove JSS and JSS theme

### PR DESCRIPTION
**Pull request by dtomaszewski88**

### CODE REVIEW

File: src/hooks/useScreenSize.tsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import { useEffect, useState } from 'react';
-import { useTheme } from '../components/Theme';
+// import { useTheme } from '../components/Theme/index123';
```
Comment (AlexP-anec) [inline src/hooks/useScreenSize.tsx]: shouldn't this be deleted ?!  :D

File: src/components/Button/Showcase.stories.tsx [hunk 2]
```diff
@@ -13,7 +13,6 @@ interface IButtonShowcase {
 
 const ButtonShowCase: React.FC<IButtonShowcase> = (props) => {
   const { sizes, shapes, colors } = props;
-  console.log('props', props);
```
Comment (AlexP-anec) [inline src/components/Button/Showcase.stories.tsx:16]: delete maybe :D

File: src/hooks/useScreenSize.tsx (hunk 1 above)
Comment (dtomaszewski88) [reply src/hooks/useScreenSize.tsx]: Done

File: src/components/Button/Showcase.stories.tsx (hunk 2 above)
Comment (dtomaszewski88) [reply src/components/Button/Showcase.stories.tsx:16]: Done

**Review by AlexP-anec — APPROVED** [review]

**Pull request merged**